internal/component/discovery/discovery.go

```diff
@@ -257,3 +273,5 @@ func (c *Component) runDiscovery(ctx context.Context, d DiscovererWithMetrics) {
 		}
 	}
 }
+
+func (c *Component) LiveDebugging(_ int) {}
```

Comment:
My understanding is that this function is triggered on a change of livedebug consumers. Should the function Publish the cached targets data as discovery may not update rapidly? CC @wildum